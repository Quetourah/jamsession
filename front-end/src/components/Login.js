
import React from 'react';


import 'bootstrap/dist/css/bootstrap.min.css';
import {Navbar,Nav, Form,InputGroup,FormControl,Button,Card} from 'react-bootstrap';

function Login () {

return(
<Navbar  className="bg-light justify-content-end">



  <Form inline pullRight >
    <InputGroup>
      <InputGroup.Prepend>
        <InputGroup.Text id="basic-addon1">@</InputGroup.Text>
      </InputGroup.Prepend>
      <FormControl
        placeholder="Username"
        aria-label="Username"
        aria-describedby="basic-addon1"
      />
    </InputGroup>
    <InputGroup>
      
      <FormControl
        placeholder="Password"
        aria-label="Password"
        aria-describedby="basic-addon1"
      />
    </InputGroup>
    <Button type="submit">Log In</Button>
  </Form>
  
  
  
</Navbar>
);
}

export default Login;
