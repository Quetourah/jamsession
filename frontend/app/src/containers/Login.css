
@media all and (min-width: 480px) {
  .Login {
    background-image: url("https://images.unsplash.com/photo-1510279410431-2d0808d69bf8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80") no-repeat fixed center center;
  }
    .Login {
      padding: 60px 0;
    }
  
    .Login form {
      margin: 0 auto;
      max-width: 320px;
      color: grey;
    }
  }