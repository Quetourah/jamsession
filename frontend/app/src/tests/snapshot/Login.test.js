import React from 'react';
import Login from '../../containers/Login';
import renderer from 'react-test-renderer';

it('renders login page correctly', () => {
  const tree = renderer
    .create(<Login/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});


