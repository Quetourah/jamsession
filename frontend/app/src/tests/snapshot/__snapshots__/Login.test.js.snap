// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders login page correctly 1`] = `
<div
  className="Login"
>
  <form
    className="form"
    onSubmit={[Function]}
  >
    <div
      className="form-group form-group-lg"
    >
      <label
        className="control-label"
        htmlFor="username"
      >
        Username
      </label>
      <input
        autoFocus={true}
        className="form-control"
        id="username"
        name="username"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-group form-group-lg"
    >
      <label
        className="control-label"
        htmlFor="password"
      >
        Password
      </label>
      <input
        className="form-control"
        id="password"
        onChange={[Function]}
        type="password"
        value=""
      />
    </div>
    <button
      className="LoaderButton button btn btn-lg btn-default btn-block"
      disabled={true}
      type="submit"
    >
      Login
    </button>
  </form>
</div>
`;
